lua/telescope/make_entry.lua

```diff
@@ -240,6 +240,22 @@ do
   end
 end
 
+function make_entry.gen_from_git_stash()
+  return function(entry)
+    if entry == "" then
+      return nil
+    end
+    local splitted = vim.split(entry, ':')
+    return {
+      value = splitted[1],
+      ordinal = splitted[2],
+      msg = splitted[2],
```

Comment:
Do we even need the msg entry somewhere? Otherwise we can just remove that tbl entry, can't we?

Reply:
remove msg entry also show git stash message in results.